README.md

```diff
@@ -114,3 +114,32 @@ If you choose to build the universal framework yourself, you can do so by runnin
 ### Contributing
 Please see [CONTRIBUTING](CONTRIBUTING.md).
 
+### Credits
+
+First-party code (under OptimizelySDKCore/, OptimizelySDKDatafileManager/, OptimizelySDKEventDispatcher/, OptimizelySDKiOS/, OptimizelySDKShared/, OptimizelySDKTVOS/, OptimizelySDKUniversal/, OptimizelySDKUserProfile/, OptimizelySDKUserProfileService/, ) is copyright Optimizely, Inc. and contributors, licensed under Apache 2.0.
```

Comment:
We no longer have code under `OptimizelySDKUserProfile`. I think we deprecated that in favor of `OptimizelySDKUserProfileService`